app/views/users/new.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %fieldset
+        = form_for @user, html: {role: "form", class: "form-horizontal"} do |f|
```

Comment:
The reason this doesn't work right is horizontal forms are done with `layout: horizontal` now.